docs/dev_guide/contents/tests.rst

```diff
@@ -87,6 +87,15 @@ or::
 
     $ pytest --remote-data=any
 
+
+Mocked remote data tests
+------------------------
+Some of our remote data tests are run using `vcrpy <https://vcrpy.readthedocs.io/en/latest/>`_, which saves a copy of any HTTP requests made within the test so a remote request doesn't have to be run on subsequent test runs.
+To use ``vcrpy`` on a test add the ``@pytest.mark.vcrpy()`` decorator to the test, and **keep the ``@pytest.mark.remote_data`` decorator**.
```

Comment:
Does it need  the `()`? I would like to avoid if it we can.

Reply:
Any reason apart for aesthetic reasons? I just copied the `pytest-vcr` docs.

Reply:
Purely aesthetic reason, it would match the our other decorators that have no args, mainly the remote data one.

Reply:
I'm going to leave the brackets as it's what's shown in the `pytest-vcrpy` docs.